src/hooks/constants/lib/fetchConstants.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { ApiPromise } from '@polkadot/api';
+import { Constants } from '../../../generated/graphql';
+import { fetchConstants } from './fetchConstants';
+import {
+  getMockApiPromise,
+  mockedRepayFee,
+} from '../../polkadotJs/tests/mockUsePolkadotJsContext';
+
+describe('fetch chain constants', () => {
+  let mockApiInstance: ApiPromise;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mockApiInstance = getMockApiPromise();
+  });
+
+  it('can retrieve lbp.getRepayFee constant', () => {
+    mockApiInstance.createType = jest
+      .fn()
+      .mockImplementation(() => ({ toHuman: jest.fn(() => mockedRepayFee) }));
```

Comment:
mockedRepayFee should be localized, please take it out of the `polkadotJs` test files.

Reply:
analysis+solution:  https://github.com/galacticcouncil/Basilisk-ui/pull/109/files#r810223042